File: DESIGN-CSharp/Command/KitchenWorker.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Command
{
    public abstract class KitchenWorker
    {
        //完成訂單
        public abstract void finishOrder();
    }

    //飲料師
    public class Barkeep : KitchenWorker 
    {
        public override void finishOrder() 
        {
            Console.WriteLine("Complete Baverage");
        }
    }

    //廚師
    public class Chef : KitchenWorker
    {
        public override void finishOrder()
        {
            Console.WriteLine("Complete Food");
        }
    }
}

File: BehavioralPatterns/Command/KitchenWorker.py
from abc import ABC, abstractmethod

class KitchenWorker(ABC):
    @abstractmethod
    def finishOrder(self):
        pass

class Barkeep(KitchenWorker):
    def finishOrder(self):
        print("Complete Baverage")

class Chef(KitchenWorker):
    def finishOrder(self):
        print("Complete Food")
